src/index.html

```diff
@@ -17,11 +17,388 @@
+
+        <h1 class="card__product-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <h3 class="card__product-id">Product code: 195434</h3>
+
+        <div class="card__reviews">
+          <div class="card__stars">
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+          </div>
+          <p class="card__reviews-label">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-label card__price-label--1">Price:</p>
+          <p class="card__price-label card__price-label--2">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image"></div>
+
+        <h1 class="card__product-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <h3 class="card__product-id">Product code: 195434</h3>
+
+        <div class="card__reviews">
+          <div class="card__stars">
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+          </div>
+          <p class="card__reviews-label">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-label card__price-label--1">Price:</p>
+          <p class="card__price-label card__price-label--2">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image"></div>
+
+        <h1 class="card__product-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <h3 class="card__product-id">Product code: 195434</h3>
+
+        <div class="card__reviews">
+          <div class="card__stars">
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+            <div class="card__star"></div>
+          </div>
+          <p class="card__reviews-label">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-label card__price-label--1">Price:</p>
+          <p class="card__price-label card__price-label--2">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
Your cards have a lot of repetitive code. Consider using a template engine like EJS or Pug to create a reusable card component. This will make your code more maintainable and easier to read.